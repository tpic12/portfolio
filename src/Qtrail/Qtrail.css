@font-face {
  font-family: "PixelFont";
  src: local("PixelFont"), url("../Fonts/PixelFont.ttf") format("truetype");
}

.extra-tech ul {
  list-style-type: none;
  display: flex;
  flex-wrap: wrap;
  padding: 0;
}

.extra-tech ul li {
  border: 1px solid rgb(62, 72, 117);
  border-radius: 3px;
  padding: 4px;
  margin: 2px 6px;
}

.links a {
  text-decoration: none;
  color: rgb(62, 72, 117);
}

.links a:hover {
  font-weight: 700;
  color: rgb(189, 89, 52);
}

.qtrail-section {
  width: 100vw;
  min-height: 100vh;
  background-color: white;
  color: rgb(62, 72, 117);
  display: flex;
  flex-direction: column;
  /* margin-top: 100px; */
  justify-content: flex-start;
  align-items: center;
  padding: 0 15px;
}

.qtrail-section h2 {
  padding-top: 100px;
  font-family: "PixelFont";
  margin-bottom: 0;
}

.logo-arrows {
  display: flex;
  flex-direction: row;
  justify-content: space-evenly;
  width: 100%;
  align-items: center;
}

.logo-arrows a {
  text-decoration: none;
  color: #ededed;
}

.links p {
  padding: 0;
}

.qtrail-circle {
  width: 100vw;
  height: 375px;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  border: 7px solid #ededed;
  border-radius: 200px;
  margin: 0 10px;
  background-color: rgb(62, 72, 117);
  color: white;
  text-align: center;
  padding: 0 28px;
}

.qtrail-circle button {
  background-color: transparent;
  border: none;
  cursor: pointer;
}

.qtrail-circle img {
  width: 100px;
  animation: pulse 2s linear;
  animation-iteration-count: infinite;
  z-index: -1;
}

.tech-logos {
  width: 50%;
  margin-top: 10px;
  display: flex;
  justify-content: space-evenly;
}

@keyframes pulse {
  0% {
    transform: scale(1);
  }

  50% {
    transform: scale(1.1);
  }

  100% {
    transform: scale(1);
  }
}

@media screen and (min-width: 720px) {
  .test {
    max-width: 900px;
  }
}