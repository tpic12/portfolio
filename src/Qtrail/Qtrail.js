import React, { Component } from "react";
import "./Qtrail.css";

class Qtrail extends Component {
  render() {
    return (
      <section className="qtrail-section snap">
        <div className="qtrail-text">
          <h2>QuaranTrail</h2>
          <p>
            In this playful and lighthearted game, the world was struck with an
            awful pandemic and the player has to stay indoors. The goal is to
            practice social distancing and self quarantine while keeping one's
            boredom and infection level bar low for as many days as possible.
          </p>
        </div>
        <div className="links">
          <p>
            <a
              href="https://quarantrail.now.sh/"
              target="_blank"
              rel="noopener noreferrer"
            >
              Live Link
            </a>{" "}
            |{" "}
            <a
              href="https://github.com/tpic12/Quarantrail-client"
              target="_blank"
              rel="noopener noreferrer"
            >
              GitHub
            </a>
          </p>
        </div>
        <div className="tech-stack">
          <h3>Technology Used:</h3>
          <ul className="tech-stack-list">
            <li>React</li>
            <li>React Game Engine</li>
            <li>Express</li>
            <li>Javascript</li>
            <li>Node</li>
            <li>PostgreSQL</li>
          </ul>
        </div>
        <div className="screenshots">
          <p>screenshots here</p>
          <p>screenshots here</p>
          <p>screenshots here</p>
        </div>
      </section>
    );
  }
}

export default Qtrail;
/**
 * As such, the two
   guages to keep an eye on are the chance of infection and boredom.
   Every choice and action can increase or decrease these guages so
   tread carefully. The player must also need to keep track of, and
   maintain, their supply of food and toilet paper, but you may not
   want to get too much...
 */
