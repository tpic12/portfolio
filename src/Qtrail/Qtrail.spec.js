import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Qtrail from "./Qtrail";

describe("<Landing />", () => {
  it("renders by default", () => {
    const wrapper = shallow(<Qtrail />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
