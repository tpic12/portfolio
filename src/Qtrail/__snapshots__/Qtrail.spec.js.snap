// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing /> renders by default 1`] = `
<section
  className="qtrail-section snap"
  id="qtrail-section"
>
  <h2>
    QuaranTrail
  </h2>
  <div
    className="links"
  >
    <p>
      <a
        href="https://quarantrail.now.sh/"
        rel="noopener noreferrer"
        target="_blank"
      >
        Live Link
      </a>
       
      |
       
      <a
        href="https://github.com/tpic12/Quarantrail-client"
        rel="noopener noreferrer"
        target="_blank"
      >
        GitHub
      </a>
    </p>
  </div>
  <div
    className="qtrail-circle"
  >
    <div
      className="logo-arrows"
    >
      <a
        href="#project-landing-section"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon="chevron-left"
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size="lg"
          spin={false}
          swapOpacity={false}
          symbol={false}
          title=""
          transform={null}
        />
      </a>
      <button
        onClick={[Function]}
      >
        <img
          alt="quaran trail - stay home"
          src="qtrail-logo.png"
        />
      </button>
      <a
        href="#linus-section"
      >
        <FontAwesomeIcon
          border={false}
          className=""
          fixedWidth={false}
          flip={null}
          icon="chevron-right"
          inverse={false}
          listItem={false}
          mask={null}
          pull={null}
          pulse={false}
          rotation={null}
          size="lg"
          spin={false}
          swapOpacity={false}
          symbol={false}
          title=""
          transform={null}
        />
      </a>
    </div>
    <p>
      In this playful and lighthearted game, the world was struck with an awful pandemic and the player has to stay indoors. The goal is to practice social distancing and self quarantine while keeping one's boredom and infection level bar low for as many days as possible.
    </p>
  </div>
  <div
    className="tech-logos"
  >
    <FontAwesomeIcon
      border={false}
      className=""
      fixedWidth={false}
      flip={null}
      icon={
        Array [
          "fab",
          "react",
        ]
      }
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size="2x"
      spin={false}
      swapOpacity={false}
      symbol={false}
      title=""
      transform={null}
    />
    <FontAwesomeIcon
      border={false}
      className=""
      fixedWidth={false}
      flip={null}
      icon={
        Array [
          "fab",
          "css3",
        ]
      }
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size="2x"
      spin={false}
      swapOpacity={false}
      symbol={false}
      title=""
      transform={null}
    />
    <FontAwesomeIcon
      border={false}
      className=""
      fixedWidth={false}
      flip={null}
      icon={
        Array [
          "fab",
          "node-js",
        ]
      }
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size="2x"
      spin={false}
      swapOpacity={false}
      symbol={false}
      title=""
      transform={null}
    />
    <FontAwesomeIcon
      border={false}
      className=""
      fixedWidth={false}
      flip={null}
      icon={
        Array [
          "fas",
          "database",
        ]
      }
      inverse={false}
      listItem={false}
      mask={null}
      pull={null}
      pulse={false}
      rotation={null}
      size="2x"
      spin={false}
      swapOpacity={false}
      symbol={false}
      title=""
      transform={null}
    />
  </div>
  <div
    className="extra-tech"
  >
    <ul>
      <li>
        React Game Engine
      </li>
      <li>
        Express
      </li>
      <li>
        Mocha
      </li>
      <li>
        Chai
      </li>
      <li>
        Jest
      </li>
    </ul>
  </div>
</section>
`;
